deadline_cloud_extension/DeadlineCloud.pyp

```diff
@@ -4,12 +4,7 @@ import sys
 root = os.path.dirname(__file__)
 sys.path.insert(0, os.path.join(root, 'modules'))
 
-if 'cinema4d_submitter' not in sys.modules.keys():
-    python_path = os.getenv('DEADLINE_CLOUD_PYTHONPATH')
-    python_paths = python_path.split(os.pathsep)
-    for n in python_paths:
```

Comment:
I was testing this and I got errors like these 

```
Traceback (most recent call last):
  File "D:\C4D_Submitter_Externsion\DeadlineCloud.pyp", line 8, in <module>
import deadline.cinema4d_submitter
ModuleNotFoundError:No module named 'deadline'
```

Reply:
Have you installed the modules to a supported interpreter path? https://developers.maxon.net/docs/py/2024_0_0a/manuals/manual_py_libraries.html#python-interpreter-bound-search-paths

Reply:
Hmm, using the `C4DPYTHONPATH311` fixed my issues.